### DISCUSSION TITLE
Equivalent of assertTableActionHalted for cancel?

### DISCUSSION THREAD

**Discussion by bianchi** [filamentphp/filament#12573] [Help]
### Package

Other

### Package Version

v3.2.72

### How can we help you?

If I halt an action using $action->halt() I can assert that in tests using `->assertTableActionHalted('my-action')`

But how can I do the same if I use $action->cancel()? There is no `->assertTableActionCanceled('my-action')`

**Comment by andrewdwallo** [comment]
Is there a way to do this? It would be nice to have this.

**Comment by howdu** [comment]
Did `assertTableActionHalted` work correctly for you guys? It seems to only check if the action is mounted.

**Reply by andrewdwallo** [reply]
The equivalent of "assertTableActionCancelled" would be "assertTableActionNotMounted", considering that cancelling an action is the same as unmounting it.
